### ISSUE TITLE
Failure running init

### ISSUE THREAD

**Issue by mfikes** [dmotz/natal#30]
This is with Natal 0.1.8 (0.1.7 works fine on this box):

```
$ natal init FutureApp
Creating FutureApp

Creating Leiningen project
Updating Leiningen project
Creating React Native skeleton
Command failed: node -e "process.argv[3]='FutureApp'; require('react-native/local-cli/init')('.', 'FutureApp')"
```

Diagnosing:

```
$ cd future-app/native
$ node -e "process.argv[3]='FutureApp'; require('react-native/local-cli/init')('.', 'FutureApp')"
module.js:339
    throw err;
    ^

Error: Cannot find module 'react-native/local-cli/init'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at [eval]:1:30
    at Object.exports.runInThisContext (vm.js:54:17)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:435:26)
    at node.js:578:27
    at doNTCallback0 (node.js:419:9)
```

**Comment by drapanjanas** [comment]
With react-native version 0.14.2 looks like this can be fixed by changing natal.coffee "node -e" command to:

```
node -e
           \"require('react-native/local-cli/cli').init('.', '#{projName}')\"
```

**Comment by dmotz** [comment]
Thanks. Not sure how I botched this and left that fix out. Should be corrected in 0.1.10.
